pages/instructorAssessmentInstances/instructorAssessmentInstances.ejs

```diff
@@ -234,6 +234,15 @@
                   data-class="text-center align-middle text-wrap"
                   data-formatter="listFormatter"
                   data-switchable="true">Group Member Names</th>
+              <th data-field="group_roles"
+                  data-sortable="true"
+                  data-sorter="roleSorter"
```

Comment:
I believe the `roleSorter` function is leftover from a previous version of this code and is no longer in place. Probably should be removed. Also, since this field will be a list, you probably should consider using a listFormatter, particularly to consider cases of groups where all students left the group (empty), or groups where all students dropped the course (None role).